woocommerce/woocommerce
# Set low stock notification threshold per product

### All Submissions:


### Changes proposed in this Pull Request:

Adds a "low stock threshold" field on the inventory tab of the edit product page.  This overrides the global value for sending out low stock notifications.  By default, the field is populated with the global low stock threshold.
Original request: http://ideas.woocommerce.com/forums/133476-woocommerce/suggestions/3590021-set-low-stock-notification-threshold-per-product

### How to test the changes in this Pull Request:

1. Make sure "enable stock management" and  "low stock notifications" are enabled in WooCommerce->Settings->Products->Inventory
2. Set a product (simple, variable/variation) to managing stock, set a low stock threshold and update
3. Purchase the product to reduce stock count and check for email notifications

### Other information:

Parent variables, if set to managing stock, will control these notifications for child variations if they are not set to managing stock.


### Changelog entry
- Option has been added on the inventory tab of the edit product page to set a low stock threshold for low stock admin notifications
